### PR TITLE
External Pledges Spanish Translation

### PR DESCRIPTION
**Pull request by lomky**
I rule and have completed some work on Case Manager that's ready for review!

(brief, plain english overview of your changes here)

This pull request makes the following changes:
* External Pledges!

screenshot from 2019-02-16 21-31-53

It relates to the following issue #s: 
* Finishes #1582

### CODE REVIEW

**Comment by lomky** [comment]
@montanezp another for Spanish review! :cat2:

File: config/locales/es.yml [hunk 1]
```diff
@@ -48,6 +48,17 @@ es:
         password: Contraseña
         password_signin: 'O, inicie sesión con una contraseña:'
         password_submit: Inicia sesión con contraseña
+  external_pledge:
```
Comment (colinxfleming) [inline config/locales/es.yml]: Love the organization of this

File: config/locales/es.yml [hunk 2]
```diff
@@ -1,5 +1,13 @@
 ---
 es:
+  external_pledge:
+    edit:
+      fund_pledge: "%{source} promesa"
+    new:
+      title: "O, grabe un nueva promesa externa:"
+      source: Fuente
```
Comment (montanezp) [inline config/locales/es.yml]: Origen

File: config/locales/es.yml [hunk 3]
```diff
@@ -55,6 +55,17 @@ es:
     pledged: envió una promesa de $%{pledge_amount}
     reached_patient: alcanzó el paciente
     unknown_action: habló o interactuó con
+  external_pledge:
+    edit:
+      fund_pledge: "%{source} promesa"
+    new:
+      amount: Cantidad
+      create_button: Crear promesa externo
+      source: Fuente
```
Comment (montanezp) [inline config/locales/es.yml]: Origen

**Pull request merged**